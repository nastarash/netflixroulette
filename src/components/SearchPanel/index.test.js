import React from 'react';
import { SearchPanel } from './index';

const defaultRoutingProps = {
  location: {
    pathname: '/',
  },
};

describe('<SearchPanel />', () => {
  test('should be rendered correctly', () => {
    const panel = shallow(
        <SearchPanel {...defaultRoutingProps} />
    );
    expect(panel).toMatchSnapshot();
  });
});
