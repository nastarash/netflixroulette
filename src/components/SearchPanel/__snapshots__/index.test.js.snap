// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchPanel /> should be rendered correctly 1`] = `
<Fragment>
  <h3
    className="search-panel__header"
  >
    FIND YOUR MOVIE
  </h3>
  <Route
    component={[Function]}
    path={
      Array [
        "/search/:searchString?",
        "/",
      ]
    }
  />
</Fragment>
`;
